app/assets/stylesheets/application.scss

```diff
@@ -59,6 +59,11 @@ html, body {
   background-color: white;
 }
 
+@media (max-width: 992px)
+.h-100 {
```

Comment:
Instead of changing the h-100 for everything, you might be better off just changing the vertical align so it's no longer centered. `align-items: unset !important;` should work inside the media query